WcaOnRails/spec/i18n_locale_files_spec.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+require 'rails_helper'
+require 'i18n-spec'
+
+Dir.glob(Rails.root.join('config', 'locales', '*.yml')).each do |locale_file|
+  RSpec.describe locale_file.to_s do
+    it { is_expected.to have_valid_pluralization_keys }
```

Comment:
Yes, `i18n-spec` requires that the locale file path is the describe subject (see [here](https://github.com/tigrish/i18n-spec#testing-the-validity-of-your-locale-files))